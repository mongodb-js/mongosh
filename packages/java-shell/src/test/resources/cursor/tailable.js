// before
db.coll.insertOne({"_id": 1, name: "Vasya"});
db.coll.insertOne({"_id": 2, name: "Petya"});
db.coll.insertOne({"_id": 3, name: "Lyusya"});
// command
db.coll.find().tailable();
// clear
db.coll.drop();