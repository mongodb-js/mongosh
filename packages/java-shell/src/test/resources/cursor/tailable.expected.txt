org.graalvm.polyglot.PolyglotException: Query failed with error code 2 and error message 'error processing query: ns=admin.collTree: $and